### PR TITLE
Automatically download Astra secure connect bundle

### PR DESCRIPTION
**Pull request by mpenick**
This adds new `--token` and `--database-id` flags to allow the proxy to connect to Astra w/o having to provide a secure connect bundle file. This is much more user friendly and makes the k8s configuration more concise.

Fixes: https://github.com/datastax/cql-proxy/issues/47

### CODE REVIEW

File: astra/bundle.go [hunk 1]
```diff
@@ -85,6 +94,68 @@ func LoadBundleZipFromPath(path string) (*Bundle, error) {
...
+	}
+
+	return LoadBundleZip(reader)
+}
+
+func readAllWithTimeout(r io.Reader, ctx context.Context) (bytes []byte, err error) {
+	ch := make(chan struct{})
+
+	go func() {
+		bytes, err = ioutil.ReadAll(r)
```
Comment (mpenick) [inline astra/bundle.go:126]: The error needs to be returned. Make `ch` an `error` channel and return that way.

**Comment by mpenick** [comment]
Might need to rework because of changes in https://github.com/datastax/cql-proxy/pull/71

**Comment by mpenick** [comment]
Tested manually against an existing Astra cluster. I've created a follow up issue to add automated testing for this: https://github.com/datastax/cql-proxy/issues/73

That will either mock out the Astra API or I will create an integration test with an existing Astra cluster in prod and have this run in CI with secrets. Maybe both?

**Review by joao-r-reis — APPROVED** [review]
Small and simple PR, looks good!

**Pull request merged**